circle.yml

```diff
@@ -17,3 +17,6 @@ deployment:
     commands:
       - yarn run build-docs
       - ./scripts/deploy-website.sh
+test:
+  pre:
+    - yarn run flow
```

Comment:
@ericvicenti you mean comments on code like the react native repo does? there's a bot for that afaik. maybe we can copy it.